content/docs/dev-and-releng/repository/app/_index.md

```diff
@@ -0,0 +1,99 @@
+---
+linkTitle: App
+title: Creating a new app repository
+description: The canonical way to create a repository for an app to be distributed and deployed via the Giant Swarm app platform.
+---
+
+These instructions will help you create a new app repository based on our [template-app](https://github.com/giantswarm/template-app).
+
+## Prerequisites
+
+- the `git` CLI
+- the [GitHub CLI](https://cli.github.com/) `gh`
+- [`devctl`](https://github.com/giantswarm/devctl) version 6.12.0 or newer
+
+## Step 1 - Preparation
+
+Since you will need the app and repository name several times on the command line, we first create an environment variable `APP_NAME` with the new repository name as the value.
+
+Example:
+
+```nohighlight
+export APP_NAME=test-app
+```
+
+Here we assume you want to create the new app and repository named `test-app`.
+
+## Step 2 - Repo creation
+
+In your current command line / shell session, navigate to the directory where you keep git clones.
+
+**Note:** No need to create an empty sub folder just for the new repo, as this will happen automatically in the next step.
+
+Now create the repository, using our [template-app](https://github.com/giantswarm/template-app) to pre-fill its content. Execute this command to create the new **public** repository:
+
+```nohighlight
+gh repo create \
+  --clone \
+  --public \
```

Comment:
Maybe I would use private in the example just in case someone misses this and accidentally publishes unintended stuff. You can always make it public later.

Reply:
Does private make sense for an app? Somehow I thought it has to be public.